go-controller/pkg/cluster/ha_master.go

```diff
@@ -0,0 +1,321 @@
+package cluster
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/ovn"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+	"github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/leaderelection"
+	"k8s.io/client-go/tools/leaderelection/resourcelock"
+)
+
+const (
+	invalidIPAddres  = "192.0.2.254"
```

Comment:
There is no guarantee that this is an invalid IP address... How about 0.0.0.1?

Reply:
Sure.

I took the invalid ip thing from here - https://github.com/ovn-org/ovn/blob/master/utilities/ovndb-servers.ocf#L45. It is refering to RFC 5737. I haven't read this RFC. But I am happy to change the invalid ip address here and probably need to revisit and change it in the OVN pacemaker script as well.